online-banking-cz/src/main/res/values/dimens.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="utf-8"?><!--
+  ~ Copyright (c) 2022 Adyen N.V.
+  ~
+  ~ This file is open source and available under the MIT license. See the LICENSE file for more info.
+  ~
+  ~ Created by atef on 25/8/2022.
+  -->
+
+<resources>
+
+    <dimen name="termsAndConditions_marginStart">17dp</dimen>
```

Comment:
Please stick to the 8 pixel grid, so it should be 16dp. Figma is sometimes wrong in this